.github/ISSUE_TEMPLATE/workflows/auto-comment.yml

```diff
@@ -0,0 +1,58 @@
+name: Auto Comment
+on:
+  issues:
+    types:
+      - opened
+      - closed
+      - assigned
+  pull_request:
+    types:
+      - opened
+      - closed
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Auto Comment on Issues Opened
+        uses: wow-actions/auto-comment@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          issuesOpened: |
+            Hey @{{ author }},
+
+            Thank you for raising an issue. We will investigate into the matter and get back to you as soon as possible.
+
+            Please make sure you have given us as much context as possible.
+
+      - name: Auto Comment on Issues Closed
+        uses: wow-actions/auto-comment@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          issuesClosed: |
+            Hey @{{ author }}, This issue is closed.
```

Comment:
Can we also show the reason of issue closing?

Reply:
yes we can Like If sone Who is not working on the issue then the issue is automatically closed then we can say that you are not working on that